src/style.css

```
@@ -1 +1,41 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```
You can unite all selectors